src/components/Nav/Nav.js

```diff
@@ -1,21 +1,41 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
-import styles from './Nav.module.css';
+import { Link, useLocation } from 'react-router-dom';
 
-const Nav = () => (
-  <nav className={styles.nav}>
-    <Link to="/">
-      <h1>Bookstore</h1>
-    </Link>
-    <ul className={styles['nav-links']}>
-      <li>
-        <Link to="/">Books</Link>
-      </li>
-      <li>
-        <Link to="/categories">Categories</Link>
-      </li>
-    </ul>
-  </nav>
-);
+const Nav = () => {
+  const location = useLocation();
+  return (
+    <nav className="px-lg py-5 bg-neutral-50 border flex gap-10 items-baseline">
+      <Link to="/">
+        <h1 className="text-blue-600 text-3xl font-bold">Bookstore</h1>
+      </Link>
+      <ul className="flex gap-10 text-xs tracking-widest">
+        <li>
+          <Link
+            to="/"
+            className={`hover:text-blue-500 ${
+              location.pathname === '/' || location.pathname === '/bookstore'
+                ? 'text-neutral-900'
+                : 'text-gray-400'
+            }`}
+          >
+            BOOKS
+          </Link>
+        </li>
+        <li>
+          <Link
+            to="/categories"
+            className={`hover:text-blue-500 ${
+              location.pathname === '/categories'
+                ? 'text-neutral-900'
+                : 'text-gray-400'
+            }`}
+          >
+            CATEGORIES
+          </Link>
+        </li>
+      </ul>
+    </nav>
```

Comment:
- [ ] In the navbar you are missing an icon on your right 
- [ ] I suggest to use this color `#0290ff` it can be a little bit like the one in the design
- [ ] Please increase a little bit your `font-size` to make your page easily readable